cl/opinion_page/templates/docket_tabs.html

```diff
@@ -184,7 +184,7 @@ <h2>{{ docket.court }}</h2>
         <span class="meta-data-value">{{ docket.date_argued }}</span>
       </p>
     {% endif %}
-
+    {% include "includes/citation.html" with obj=docket fmt=cite_format %}
```

Comment:
Where is `cite_format` coming from or being used?

Reply:
Moved to cl.settings.django as DEFAULT_CITE_FORMAT

Reply:
We definitely don't need this as a global setting, since it's not something that I expect will be used in many places. It'd be much better to put it at the top of this file or even just in this function until it's proved that it's needed somewhere else.